org.openhab.binding.zigbee.ember/ESH-INF/thing/controller_ember.xml

```diff
@@ -8,100 +8,160 @@
+				<label>Reset Controller</label>
+				<description>Resets the Controller and sets the configuration to the configured values.</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="zigbee_channel" type="integer" groupName="network" required="false" min="11" max="25">
+				<label>Channel</label>
+				<description>Channel number</description>
+				<default>-1</default>
+				<options>
+					<option value="-1">Auto</option>
+					<option value="11">Channel 11</option>
+					<option value="12">Channel 12</option>
+					<option value="13">Channel 13</option>
+					<option value="14">Channel 14</option>
+					<option value="15">Channel 15</option>
+					<option value="16">Channel 16</option>
+					<option value="17">Channel 17</option>
+					<option value="18">Channel 18</option>
+					<option value="19">Channel 19</option>
+					<option value="20">Channel 20</option>
+					<option value="21">Channel 21</option>
+					<option value="22">Channel 22</option>
+					<option value="23">Channel 23</option>
+					<option value="24">Channel 24</option>
+					<option value="25">Channel 25</option>
+				</options>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="zigbee_panid" type="Integer" groupName="network" required="false">
+				<label>PAN ID</label>
+				<description>PAN Network ID</description>
+				<default>0</default>
+				<options>
+					<option value="65535">Auto</option>
+				</options>
+				<limitToOptions>false</limitToOptions>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="zigbee_extendedpanid" type="text" groupName="network" required="false">
+				<label>Extended PAN ID</label>
+				<description>Extended PAN Network ID: 16 byte hexadecimal value</description>
+				<default>0000000000000000</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="zigbee_networkkey" type="text" groupName="network">
+				<label>Network Security Key</label>
+				<description>Set the network security key</description>
+				<default>00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00</default>
+				<advanced>true</advanced>
+			</parameter>
+
+		</config-description>
+
 	</bridge-type>
-	
+
+	<channel-type id="ember_ash_rx_dat">
+		<item-type>Number</item-type>
+		<label>Data frames received</label>
+		<description>Provides a count of DATA frames received by the host.</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+
+	<channel-type id="ember_ash_tx_dat">
+		<item-type>Number</item-type>
+		<label>Data frames sent</label>
+		<description>Provides a count of DATA frames sent by the host.</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+
+	<channel-type id="ember_ash_rx_nak">
+		<item-type>Number</item-type>
+		<label>NAK frames received</label>
+		<description>Provides a count of DATA frame with an error or that were discarded by the NCP due to lack of memory.
```

Comment:
Minor comment: I think that here, and in the channel types below, it should be `DATA frames` (and not `DATA frame`).